src/features/AppBar/AppBar.tsx

```diff
@@ -12,24 +12,24 @@ import { getAuth, signOut } from 'firebase/auth';
 import Typography from '@mui/material/Typography';
 import MUIAppBar from '@mui/material/AppBar';
 import Toolbar from '@mui/material/Toolbar';
-import useScrollTrigger from '@mui/material/useScrollTrigger';
+// import useScrollTrigger from '@mui/material/useScrollTrigger';
```

Comment:
Not using this at the moment because of the layout of the dashboard. But can be used on other pages such as the main page or the information pages that will be created soon